src/pytest_postgresql/plugin.py

```diff
@@ -96,6 +97,12 @@ def pytest_addoption(parser):
         default='tests'
     )
 
+    parser.addini(
```

Comment:
https://docs.pytest.org/en/latest/reference.html?highlight=addoption#pytest.config.argparsing.Parser.addini doesn't this needs a type declaration as pathlist?

Reply:
In theory, I've just pushed a fix for that.